### PR TITLE
Chore: upadated documentation

### PR DESCRIPTION
**Pull request by guacamole**
Signed-off-by: guacamole <gunjanwalecha@gmail.com>

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
 module.exports = {
-  //base: 'https://blog.openregistry.dev'
+  //base: 'https://docs.openregistry.dev'
   base: '/',
-  head: [['link', { rel: 'icon', href: 'https://parachute.openregistry.dev/static/media/parachute.4575caeb.png'}]],
+  head: [['link', { rel: 'icon', href: 'https://avatars.githubusercontent.com/u/90045228?v=4'}]],
```
Comment (jay-dee7) [inline docs/.vuepress/config.js]: @guacamole add someting like this here: https://raw.githubusercontent.com/containerish/docs/main/assets/parachute.png
don't worry if it doesn't exist already, just make sure to include it in the PR

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-  title: "Open Registry Blog",
+  title: "Blog",
   description: "A decentralised container registry fully compliant with OCI Distribution Specification",
 
   themeConfig: {
     darkMode: true,
     repo: 'https://github.com/containerish',
     editLink: false,
-    logo: 'https://parachute.openregistry.dev/static/media/parachute.4575caeb.png',
+    logo: 'https://app.openregistry.dev/logo.svg',
```
Comment (jay-dee7) [inline docs/.vuepress/config.js]: and here too

Comment (guacamole) [reply docs/.vuepress/config.js]: alright!

File: docs/.vuepress/config.js (hunk 1 above)
Comment (guacamole) [reply docs/.vuepress/config.js]: 👍

**Review by jay-dee7 — APPROVED** [review]
@guacamole LGTM 🥇 🥇 🥇 🥇

**Pull request merged**